server/routes/contacts/create/createContactMiddleware.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, RequestHandler } from 'express'
+
+const ensureInCreateContactJourney = (): RequestHandler => {
+  return async (req: Request<{ journeyId: string }, unknown, unknown>, res, next) => {
```

Comment:
I'm planning to add some constraints on the number of journeys as per CSIP but will do that in a separate PR.
Also need to think about the journey entry points, they probably need to be encoded in the URL somewhere to allow us to restart properly (or perhaps we just error?)

Reply:
If they're inside one journey, but paste in a URL, or type one, then yes I think we'd need to recognise this and ensure they can't jump into a later part of the same journey or a different journey altogether.